### PR TITLE
changed the font-size [issue 7]

### LINKED ISSUE #7: Decrease font size
decrease font size for higher value blocks, so that it doesn't touch the sides/goes out the box

### PR DESCRIPTION
**Pull request by lebedevhhh**
fixes #7

### CODE REVIEW

File: app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java [hunk 1]
```diff
@@ -18,10 +18,12 @@ public final class Renderer2048 {
     private static final int TILE_SIDE = 200;
     private static final int PADDING = 20;
     private static Map<Integer, Color> tileColorMap = new HashMap<>();
+    private static int fontSize = 100; //set the font size here
 
     private Game2048 game;
     private final BufferedImage image;
 
+    //runs when the class is firstly loaded for one time
```
Comment (Taz03) [inline app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java:26]: remove this comment

File: app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java [hunk 2]
```diff
@@ -18,10 +18,12 @@ public final class Renderer2048 {
     private static final int TILE_SIDE = 200;
     private static final int PADDING = 20;
     private static Map<Integer, Color> tileColorMap = new HashMap<>();
+    private static int fontSize = 100; //set the font size here
```
Comment (Taz03) [inline app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java:21]: u just made the font size smaller in general, that would work but not how we want it to. u should calculate the fontsize depending on block value

**Review by Taz03 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java (hunk 2 above)
Comment (lebedevhhh) [reply app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java:21]: kk I'll see what I can do, tyvm for the reply

File: app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java [hunk 3]
```diff
@@ -18,6 +18,9 @@ public final class Renderer2048 {
     private static final int TILE_SIDE = 200;
     private static final int PADDING = 20;
     private static Map<Integer, Color> tileColorMap = new HashMap<>();
+    private static int defaultFontSize=120;
+    private static int scale=8;
```
Comment (Taz03) [inline app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java:22]: make these final as well

**Review by Taz03 — CHANGES_REQUESTED** [review]

**Comment by Taz03** [comment]
can u show how it looks with these changes?

File: app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java (hunk 3 above)
Comment (lebedevhhh) [reply app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java:22]: yup makes sense

File: app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java [hunk 4]
```diff
@@ -18,6 +18,9 @@ public final class Renderer2048 {
     private static final int TILE_SIDE = 200;
     private static final int PADDING = 20;
     private static Map<Integer, Color> tileColorMap = new HashMap<>();
+    private static final int defaultFontSize=120;
+    private static final int scale=8;
```
Comment (Taz03) [inline app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java:22]: final vars should always be named in capital letters also move it to the top with other final vars, also add space before and after the `=`

**Review by Taz03 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java (hunk 4 above)
Comment (lebedevhhh) [reply app/src/main/java/com/togetherjava/tjplays/games/game2048/Renderer2048.java:22]: ok np

**Pull request merged**